pkgs/development/tools/trunk/default.nix

```diff
@@ -20,7 +20,7 @@ rustPlatform.buildRustPackage rec {
   # requires network
   checkFlags = [ "--skip=tools::tests::download_and_install_binaries" ];
 
-  cargoHash = "sha256-+jz0J1qFK2fZ4OX089pgNtT2vfiOTf39qQjeXmLoFNs=";
+  cargoHash = "sha256-RMB06jiuD+3uKkNPo5Mfyjl574f5IszTLgQcow0d5uI=";
```

Comment:
looks like hash vendor hash is not stable ?

error: hash mismatch in fixed-output derivation '/nix/store/00493nx00s0g596dmf4xkbh8hb3jhggh-trunk-0.18.0-vendor.tar.gz.drv':
         specified: sha256-RMB06jiuD+3uKkNPo5Mfyjl574f5IszTLgQcow0d5uI=
            got:    sha256-ikzE1ZH6ueU8yMmr3gPsgR+bG4Zfz0SMitWQdbC9NAA=

Reply:
Odd, I just retried this locally and didn't have an issue, also tried using a fakeHash to force a hash mismatch and got `sha256-RMB06jiuD+3uKkNPo5Mfyjl574f5IszTLgQcow0d5uI=`. Also questioning why Ofborg doesn't find an issue with a hash mismatch.

Reply:
sorry forgot to mention, hash mismatch happens in ofborg on x86_64-darwin

Reply:
I'll look into this

Reply:
Have fixed this in https://github.com/NixOS/nixpkgs/pull/274304/commits/0767cfe9d29fb60ee521efae9fe2bf77f0ef1043 using a conditional to give a different `cargoHash` dependent on the users architecture